src/io/video.js

```diff
@@ -126,7 +126,7 @@ class Video {
      */
     setPreviewGhost (ghost) {
         this._ghost = ghost;
-        if (this._drawable) {
+        if (this._drawable !== -1) {
```

Comment:
Could you add a comment above this line about what -1 represents (I assume this._drawable === -1 represents that the video is off, right?)